server/src/model/_cli.ts

```diff
@@ -121,51 +129,57 @@ async function lintLinks(bookDirs: string[]) {
             new URL(url)
           } catch {
             hasErrors = true
-            log(`Error: Could not parse URL '${url}' and urlEncoded to show any odd unicode characters`, encodeURI(url))
+            logCsv(bundle, page, link, 'parse', encodeURI(url))
             continue
           }
 
-          proto.get(url, res => {
-            if (res.statusCode !== undefined) {
-              if (res.statusCode >= 200 && res.statusCode < 300) {
-                console.log('Ok:', res.statusCode, link.url)
-              } else if (res.statusCode >= 300 && res.statusCode < 400) {
-                log('Following Redirect:', res.statusCode, link.url)
-                const destUrl = res.headers.location
-                if (destUrl !== undefined) {
+          let destUrl = ''
+          try {
+            const res = await fetch(url, { redirect: 'manual' })
+            if (res.status !== undefined) {
+              if (res.status >= 200 && res.status < 300) {
+                info('Ok:', res.status, link.url)
+              } else if (res.status >= 300 && res.status < 400) {
+                info('Following Redirect:', res.status, link.url)
+                destUrl = res.headers.get('location') ?? ''
+                if (destUrl !== undefined && destUrl !== null) {
                   // -------------------
                   // Avert your eyes!
                   // This is lazy copy/pasta
                   // -------------------
-                  let proto2: typeof http | typeof https = http
-                  if (destUrl.startsWith('https:')) {
-                    proto2 = https
+
+                  if (destUrl.includes('www.fs.fed.us') || destUrl.includes('news.discovery.com')) {
```

Comment:
Why is it checking for these two specific hostnames?

Reply:
Oops, fixed!